### ISSUE TITLE
Is there a "Profile" Equivalent in dia-nn Output for deepnovo-dia 

### ISSUE THREAD

**Issue by yangshu729** [vdemichev/DiaNN#796]
I am currently attempting to modify the deepnovo-dia model, which takes a "profile" as part of its input. This "profile" represents intensity values over the retention time range and is formatted as pairs of "time:intensity," separated by semicolons. The time points also align with the time of spectra in the "scans" column.

Does the output from diann contain a field that conveys the same or similar information? I haven't found it yet and would appreciate any guidance.

**Comment by vdemichev** [comment]
No, nothing similar. You can get fragment XICs usiong --vis for specific peptides, but it will only extract them for fragments in the library.

Best,
Vadim

**Comment by yangshu729** [comment]
你发的邮件我已经收到！谢谢！

**Comment by YuL-BI** [comment]
Hi Vadim,

I am trying to search DIA data with modification of Succinyl (K0, and I have added the command line in the “Additional options” box as below:
                --mod UniMod:64, 100.016044

But I got nothing at all.  The spectral library was made use the standard settings.

I got thousands of the modified peptides from the same sample by DDA.  So I wonder that I may did  something wrong in the DIA-NN setting. Could you please advise?

Well, I am very poor on programming …

Thank you very much!

Kind regards,

Lu

**Comment by vdemichev** [comment]
Hi Lu,

The spectral library needs to be made with the respective --var-mod or --fixed-mod.

Best,
Vadim

**Comment by YuL-BI** [comment]
Hi Vadim,

Do you mean that I add this when I make the spectral library:
                --var mod UniMod:64, 100.016044

Thanks very much.

Best wishes,

Lu

**Comment by vdemichev** [comment]
Yes, but with different syntax for --var-mod, please see docs on it.
